builtin/logical/ssh/path_fetch.go

```diff
@@ -23,24 +23,27 @@ func pathFetchPublicKey(b *backend) *framework.Path {
 			logical.ReadOperation: b.pathFetchPublicKey,
 		},
 
-		HelpSynopsis:    `Retrieve the public key.`,
-		HelpDescription: `This allows the public key of the SSH CA certificate that this backend has been configured with to be fetched. This is a raw response endpoint without JSON encoding; use -format=raw or an external tool (e.g., curl) to fetch this value.`,
+		HelpSynopsis:    `Retrieve the 'default' issuer's public key.`,
+		HelpDescription: `This endpoints allows fetching the configured default SSH issuer's public key. This is a raw response endpoint without JSON encoding; use -format=raw or an external tool (e.g., curl) to fetch this value.`,
 	}
 }
 
 func (b *backend) pathFetchPublicKey(ctx context.Context, req *logical.Request, data *framework.FieldData) (*logical.Response, error) {
-	publicKeyEntry, err := caKey(ctx, req.Storage, caPublicKey)
+	sc := b.makeStorageContext(ctx, req.Storage)
+
+	issuer, err := sc.fetchDefaultIssuer()
 	if err != nil {
-		return nil, err
+		return handleStorageContextErr(err)
 	}
-	if publicKeyEntry == nil || publicKeyEntry.Key == "" {
-		return nil, nil
+
+	if issuer == nil {
+		return logical.ErrorResponse("No keyuaslkdjnmaskldjaskld"), nil
```

Comment:
@Gabrielopesantos This looks unintentional?

```suggestion
		return logical.ErrorResponse("No key corresponding to issuer"), nil
```